starter-code/css/main.css

```diff
@@ -6,3 +6,278 @@ Green: #00B172
 White: #FFF
 
 */
+
+/* General styling  */
+
+html,body {
+
+height:100%;
+margin:0%;
+font-size: 24px;
+font-family: Arial, Helvetica, sans-serif;
+    
+    }
+
+h1 {
+
+font-size: 80px;
+color: white;
+
+
+}
+
+h2 {
+
+font-size: 35px;
+margin: 0px 400px 0px 400px;
+text-align: center;
+color: white;
+    
+    }
+
+h3 {
+
+font-size: 40px;
+border-bottom: 4px solid #1DB954;
+
+}    
+
+h4 {
+
+color: #1DB954;
+font-size: 30px;
+text-align: center;
+
+}
+
+h5 {
+
+color: white;
+font-size: 40px;
+border-bottom: 4px solid rgb(255, 255, 255);
+    
+    
+    }
+
+
+
+h6 {
+
+color: white;
+font-size: 30px;
+margin-bottom: 20px;
+
+
+}
+
+/* Nav bar  */
+
+
+
+div.spacer {
+
+height: 100px;
+
+
+}
+
+nav {
+
+display: flex;
+justify-content: space-between;
+position: fixed;
+height: 100px;
+background-color: white;
+width: 100%;
+align-items: center;
+padding: 0 20px 0 20px;
+
+}
+
+
+nav ul li {
+    
+display: inline;
+font-size: 25px;
+padding: 0px 10px 0px 10px;
+  
+}
+
+nav div.logo {
+
+background-image: url('/starter-code/images/spotify-logo.png');
+height: 70px;
+width: 220px;
+background-repeat: no-repeat;
+background-size: cover;
+margin-left: 30px;
+
+}
+
+nav div.navigation-items {
+
+margin: 0px 50px 0px 30px;
+
+
+}
+
+
+/* Header image section*/
+
+
+
+
+section.header-image {
+
+background-image: url('/starter-code/images/landing.jpg');
+height: 100vh;
+width:100%;
+background-repeat: no-repeat;
+background-size: cover;
+display: flex;
+flex-flow:column;
+align-items: center;
+justify-content: center;
+
+
+}
+
+
+/* USP section*/
+
+div.usp-header {
+
+display: flex;
+align-items: center;
+justify-content: center;
+    
+}
+
+
+div.usp-container {
+
+display: flex;
+flex-direction: row;
+justify-content: space-evenly;
+text-align: center;
+
+
+}
+
+div.usp-box {
+
+padding: 20px;
+margin: 20px 90px 20px 90px;
+
+}
+
+
+div.usp-box div {
+
+margin-left: auto;
+margin-right: auto;
+
+
+}
+
```

Comment:
I noticed that your classes '.loudspeaker-icon','.music-icon' and '.stream-icon' have 4 identical CSS properties. I would suggest maybe creating a single class '.icon-model' which can be used on as many icons as you need, you get less code and better structure.

Reply:
Thanks, i adjusted the paths and made a general icon class